src/coreComponents/unitTests/meshTests/testMeshEnums.cpp

```diff
@@ -36,9 +36,11 @@ TEST( MeshEnums, ElementType )
   ASSERT_EQ( "C3D5", toString( EnumType::Pyramid ) );
   ASSERT_EQ( "C3D6", toString( EnumType::Wedge ) );
   ASSERT_EQ( "C3D8", toString( EnumType::Hexahedron ) );
+  ASSERT_EQ( "C3D10", toString( EnumType::Prism5 ) );
+  ASSERT_EQ( "C3D12", toString( EnumType::Prism6 ) );
```

Comment:
Was this test useful? (I'm looking for feedback)

Reply:
Yes